lib/main.dart

```diff
@@ -36,72 +36,91 @@ class _RandomWordsState extends State<RandomWords> {
         ],
       ),
       body: _buildSuggestions(),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          _suggestions.addAll(generateWordPairs().take(5));
+          setState(() {}); // refresh page
+        },
+        child: const Icon(Icons.plus_one),
+        backgroundColor: Colors.green,
+      ),
     );
   }
 
   Widget _buildSuggestions() {
     return ListView.builder(
       padding: const EdgeInsets.all(16.0),
+      itemCount: _suggestions.length,
```

Comment:
Here you assign length of suggestions to itemCount variable but it is not using anywhere. Remove it or better use it in line 55